code/src/components/Question2.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const Question2 = ({ goal,  onGoalChange, onStepChange}) => {
+    
+    const goals = [
+        'Getting some fresh air', 
+        'Working on my skiing skills', 
+        'Having a good time with friends', 
+        'Training for the Olympics']
+
+    return (
+      <div>
+            <article className="survey-container">
+                <form className="form">
+                    <h1 tabindex="0">What are you main goal for this ski trip?</h1>
+                    <div className="goal-input-container">
+                        {goals.map(goal => (
+                            <label className="radio-button-text" key={goal} tabindex="0">
+                                <input
+                                    className="radio-button"
+                                    type="radio"
+                                    value={goal}
+                                    onChange={onGoalChange}
+                                    tabindex="0"
+                                    >
```

Comment:
I think you should add a checked to the index to fixed the button problem. 
Maybe:
checked={'type your Input' === 'type your value'} if i read it correct yours should be goal === goal. 
I hade different names for it.

Reply:
Thanks, Ida - great help!